TestSupport/planaritytesting/g6_generation_and_comparison_driver.py

```diff
@@ -230,7 +249,8 @@ def _generate_makeg_g6_files_for_order_and_num_edges(
         """Generate makeg .g6 and makeg canonical .g6 files
 
         The makeg .g6 and makeg canonical .g6 files generated have paths:
-            {self.output_parent_dir}/{order}/n{order}.m{num_edges}.makeg(.canonical)?.g6
+            {self.output_parent_dir}/{order}/
```

Comment:
That pylint though!

Reply:
Yeah, in this case I included the path to the parent directory on one line and the filename on the subsequent line because the naming convention is OUTTA CONTROLLL lol!